app/src/main/java/com/github/libretube/ui/activities/OfflinePlayerActivity.kt

```diff
@@ -91,13 +93,11 @@ class OfflinePlayerActivity : BaseActivity() {
             super.onPlaybackStateChanged(playbackState)
             // setup seekbar preview
             if (playbackState == Player.STATE_READY) {
-                binding.player.binding.exoProgress.addSeekBarListener(
-                    SeekbarPreviewListener(
-                        timeFrameReceiver ?: return,
-                        binding.player.binding,
-                        playerController.duration
-                    )
-                )
+                with(binding.player.binding.exoProgress){
+                    seekBarPreviewListener?.also { removeSeekBarListener(it) }
```

Comment:
```suggestion
                    seekBarPreviewListener?.let { removeSeekBarListener(it) }
```

Reply:
This is just for consistency, as we're using `let` everywhere else in the code for doing such operations on nullable operators .